--- Makefile	2008-05-19 21:21:39.000000000 +0200
+++ Makefile.oden	2008-08-03 19:46:46.000000000 +0200
@@ -9,6 +9,7 @@
 CC=gcc
 LIB= -lcrypt
 LOCALFLAGS= -g 
+LDFLAGS=
 
 # For PAM on Redhat Linux
 # LIB=-lpam -ldl
@@ -24,10 +25,10 @@
 	auth_pam.o auth_sun.o fail_log.o lastlog.o nologin.o snooze.o
 	$(CC) -o pwauth $(CFLAGS) main.o auth_aix.o auth_bsd.o auth_hpux.o \
 		auth_mdw.o auth_openbsd.o auth_pam.o auth_sun.o fail_log.o \
-		lastlog.o nologin.o snooze.o $(LIB)
+		lastlog.o nologin.o snooze.o $(LDFLAGS) $(LIB)
 
 checkfaillog: checkfaillog.o fail_check.o
-	$(CC) -o checkfaillog $(CFLAGS) checkfaillog.o fail_check.o $(LIB)
+	$(CC) -o checkfaillog $(CFLAGS) checkfaillog.o fail_check.o $(LDFLAGS) $(LIB)
 
 main.o: main.c config.h pwauth.h fail_log.h
 auth_aix.o: auth_aix.c config.h pwauth.h
